tests/passes/static-timing/compile-while-static.expect

```diff
@@ -0,0 +1,22 @@
+enables:
+[0, 1)
+  cond0[go] = 1'd1;
+[1, 2)
+  do_add[go] = 1'd1;
+[2, 3)
+  cond0[go] = 1'd1;
+[3, 4)
+  do_add[go] = 1'd1;
+[4, 5)
+  cond0[go] = 1'd1;
+transitions:
+(0)->(1)
+  1'b1
+(1)->(2)
+  1'b1
+(2)->(3)
+  1'b1
+(3)->(4)
+  1'b1
+(4)->(5)
+  1'b1
```

Comment:
It occurs to me that there might be a simple optimization possible in the `realize_schedule` method to calculate consecutive "runs" of FSM states where we basically just want `fsm.in = fsm.out + 1` and simplify the generated transitions a bit. Not super crucial but will probably start mattering when you look at resources.

Reply:
Opened an issue for this enhancement so we don't lose it: https://github.com/cucapra/calyx/issues/929.